vpt/__main__.py

```diff
@@ -14,5 +15,5 @@
     check()
 elif args['cmd'] is None:
     # If no argument, fall back to recording
-    # record(args['audio'], args['video'])
+    record(args['audio'], args['video'])
```

Comment:
If `cmd` is `None`, it means it didn't use any of the subparsers, which means we don't have `args['audio']` or `args['video']`. We should just use the defaults since we don't have a way of selecting them in the GUI